.github/workflows/ci.yml

```diff
@@ -55,6 +55,6 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
           TAG: ${{ inputs.tag }}
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
-        with:
-          directory: packages
+        uses: codecov/codecov-action@v4
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@tannerlinsley @TkDodo @arnoud-dv Could you add codecov token as secrets for GitHub Actions?

Reply:
thanks. I added the secret 👍